resources/RU.json

```
@@ -0,0 +1,32 @@
+{
+  "main":
+    {
+      "0": "Тебя встречает приятный молодой человек в пыльной рабочей одежде.\n\n— Привет! Казино 'test v:α.0.2.1 в стадии разработки, как видишь, — За его спиной с грохотом падает лестница и слышен отчаянный вопль кошки, — Не обращай внимание, поиграть всё ещё есть во что.\n— Хм... мы все заняты... Хотя нет, вон, смотри, это Катя. Катерина! Вот тебе кубик, катя составит тебе компанию. А мне пора, увидимся!\n",
+      "1": "В твоём инвентаре появился 1 кубик(\uD83C\uDFB2)",
+      "2": "Ты проходишь в соседнюю комнату, где потягивается на матраце Катя.\n"
+    },
+  "GameKate":
+    {
+      "0": "— Привет, я Катя. А тебя как зовут",
+      "1": ", верно? Будем знакомы ^^.\n Игра старая и очень простая: Для победы нужно бросая кубики набрать",
+      "2": "очков.\n Ты гость, поэтому я дам тебе право хода.\n Ход представляет собой серию бросков кубика неограниченное число раз и суммирование выпадающих значений.\n Однако если выпадает 1, вся сумма, набранная за ход, аннулируется и ход переходит ко оппоненту.\n Игрок может завершить свой ход в любой момент.\n",
+      "gaming_start": "Начинаем?\n",
+      "gaming_winner": "Победитель",
+      "gaming_gratitude": "Спасибо за игру. Сыграем ещё раз?",
+      "gaming_end": "Ок, тогда я пошла спать. Заходи ещё.",
+      "game_lastdot": "— Кажется, кого-то от победы отделяет 1 шаг. Ты веришь в судьбу? Сейчас узнаем на чьей она стороне.\n",
+      "try_gamer_start": "Бросает игрок. У тебя ",
```

```suggestion
      "try_gamer_start": "Бросает игрок. У тебя {} очков",
```

А  потом в коде используй формат:
``` 
RM.get_text_by_id('GameKate', 'try_player_start').format(score)
```